src/c77_menuFOC.hpp

```diff
@@ -0,0 +1,120 @@
+#pragma once
+
+#if (DISPLAY_TYPE > 0) && (FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE)
+
+// HIGHLIGHT states allow you to pick one of the sub functions.
+enum
+{
+  HIGHLIGHT_FOCUS_FIRST = 1,
+  HIGHLIGHT_FOCUS_ADJUSTMENT = 1,
+
+  HIGHLIGHT_FOCUS_LAST = HIGHLIGHT_FOCUS_ADJUSTMENT
+};
+
+#define FOCUS_ADJUSTMENT 90
+
+byte focState = HIGHLIGHT_FOCUS_FIRST;
```

Comment:
Don't use raw numbers for state machines please! :)

Reply:
Fixed